server/routes/offenceCodeDecisions/decisionAnswers.ts

```diff
@@ -0,0 +1,8 @@
+// Allows the recording of additional data when answering questions to be retained while progressing through decisions.
+export type DecisionAnswers = {
+  victimOfficer?: string
```

Comment:
Officer or staff? Aren't they the same?

Reply:
They map to different radio buttons, so they are slightly different, but at the back end it will be the same